change/@ni-nimble-components-e476eedb-989a-4246-811b-039ca8a8352e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We captured this issue in a separate work item in AzDo [here](https://dev.azure.com/ni/DevCentral/_backlogs/backlog/ASW%20SystemLink%20LIMS/Features/?workitem=2472376).

It appears that this issue is a Firefox bug that arises when a `contenteditable` div element is used within a `shadow root`. Here is the bug link: https://bugzilla.mozilla.org/show_bug.cgi?id=1496769. 

Here is a quick demonstration link to reproduce this bug: https://bug1496769.bmoattachments.org/attachment.cgi?id=9014846 
![firefox-issue](https://github.com/ni/nimble/assets/123377523/2153a868-44e6-4482-ab98-4243e072be0e)

Since this issue is not fixed yet in the Firefox browser,  it may be necessary for us to implement a workaround here in order to address this issue as it affects user experience. However, we will plan to take this in a subsequent PR. Since the current PR serves as the foundation for other features of the `nimble-rich-text-editor`, we don't want to block its progress due to this issue.

Reply:
Great research! I'd agree that a workaround may be necessary since the issue has been open for a long time (5 years) so I doubt we can expect a fix soon. I'm ok with deferring it to a future PR though.

Reply:
As mentioned, we will track this work here in AzDo: https://dev.azure.com/ni/DevCentral/_backlogs/backlog/ASW%20SystemLink%20LIMS/Features/?workitem=2472376

And, we will be creating a separate PR to write a workaround for this bug.